app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const { getTopics } = require('./controllers/topicsController')
+console.log(getTopics)
+const app = express();
+
+app.get('/api/topics', getTopics)
+
+module.exports = app;
```

Comment:
You could move this module.exports to the bottom of the file just to keep it uniform with where we keep it on other files :)